### PR TITLE
Added Flask App

### PR DESCRIPTION
**Pull request by shreya024**
Related Issue : #9

### CODE REVIEW

**Comment by hritikksingh** [comment]
@shreya024 have you done any integration testing on your local machine?

**Comment by hritikksingh** [comment]
@shreya024, please respond, or else I won't be able to merge it to the main branch

**Comment by shreya024** [comment]
> @shreya024, please respond, or else I won't be able to merge it to the main branch

I am currently having my sem exams till 10th.I had checked once in my local(it was working fine).I will surely check it again in case there r any errors after 10th.

**Comment by anandxkumar** [comment]
Screenshot from 2022-05-02 23-52-16

Issues:

- Formatting of HTML is wrong (as text isn't getting rendered properly).
- Choose file selects file but doesn't upload it, then is forward to browser in base directory to select the file which actually loads the video and plays it.

File: app.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+app = Flask(__name__)
+app.secret_key = 'some secret key'
+
+
+@app.route('/', methods=['GET'])
+def index():
+    return render_template('index.html')
+
+@app.route('/predict', methods=['GET', 'POST'])
+def upload():
```
Comment (anandxkumar) [inline app.py:17]: Upload isn't working properly

**Review by anandxkumar — CHANGES_REQUESTED** [review]
Look into the possible code causing the issues

**Pull request closed without merging**